pwscf/v6.2.1/pw2wannier90.f90

```diff
@@ -957,6 +969,45 @@ SUBROUTINE read_nnkp
           center_w(1:3,iw),l_w(iw),mr_w(iw),r_w(iw),alpha_w(iw)
   ENDDO
 
+  ! vv: Read SCDM block
+  scdm_proj_tmp = 0
+  scdm_proj = .false.
+  scdm_entanglement = 0
+  scdm_mu = 0.0_DP
+  scdm_sigma = 0.0_DP
+  IF (ionode) THEN
+     CALL scan_file_to('scdm_info',found)
+     !IF(.NOT. found) THEN
+     !   CALL errore( 'pw2wannier90', 'Could not find scdm info block in'&
+     !        &//trim(seedname)//'.nnkp',1)
+     !ENDIF
+     IF (found) THEN
+        READ (iun_nnkp,*) scdm_proj_tmp
+        IF(scdm_proj_tmp==1) THEN
+           scdm_proj = .true. 
+           READ (iun_nnkp,*) n_wannier
+           READ (iun_nnkp,*) scdm_entanglement
+           READ (iun_nnkp,*) scdm_mu, scdm_sigma
+        ENDIF
+     ELSE
+        WRITE(stdout,'(//," ****** begin WARNING ****** ",/)') 
```

Comment:
Do we really want to issue a Warning? If the user does not want to use SCDM, then he does not need to upgrade to a recent version. If he wants to, then he will have to have already upgraded. 
This warning might "scare" users but I don't think it is needed (most probably they will read this and don't know what SCDM is).